packages/ai-jsx/src/batteries/docs.tsx

```diff
@@ -600,6 +600,45 @@ export class LocalCorpus<
   }
 }
 
+/** A fully mananged {@link Corpus} served by Fixie. */
+export class FixieCorpus<ChunkMetadata extends Jsonifiable = Jsonifiable> implements Corpus<ChunkMetadata> {
+  private static readonly FIXIE_API_URL = 'https://app.fixie.ai/api';
+
+  constructor(private readonly corpus_id: string, private readonly fixie_api_key?: string) {
+    if (!fixie_api_key) {
+      this.fixie_api_key = getEnvVar('FIXIE_API_KEY');
```

Comment:
```suggestion
      this.fixie_api_key = getEnvVar('FIXIE_API_KEY', false);
```

Pass `shouldThrow = false` here since you want to handle the error yourself.